source/chapters/appendix/mixxx_controls.rst

```diff
@@ -820,6 +820,46 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
    .. versionadded:: 2.0.0
 
 
+.. mixxx:control:: [ChannelN],shift_cues_earlier
+                   [PreviewDeckN],shift_cues_earlier
+                   [SamplerN],shift_cues_earlier
+
+   :range: binary
+   :feedback: All :term:`hotcues <hotcue>` move left by 10ms.
```

Comment:
how about this?
Only downside is that we can not link to :term:.
Not sure if we should link to djing_with_mixxx#intro-and-outro-cues AND :term:cue AND :term:hotcues ?
```suggestion
   :feedback: All cue markers move left by 10ms.
```

Reply:
we could also just add a glossary entry for "cue marker" which in turn links to those?

Reply:
of course 🙄 I'd prefer that, too

Reply:
Done